WasteManagement/WasteContainer/doc/spec.md

```diff
@@ -182,7 +186,8 @@ responsible, district, neighbourhood, etc.
     + Attribute type: [DateTime](https://schema.org/DateTime)
     + Optional
     
-+ `TimeInstant` : Timestamp saved by FIWARE's IoT Agent as per dynamic IoT data arrival. Note: This attribute has not been harmonized
++ `TimeInstant` : [Timestamp](https://github.com/telefonicaid/iotagent-node-lib/blob/develop/README.md#TimeInstant)
```

Comment:
Just confirming with you that you really want to link it to the `develop` branch of the https://github.com/telefonicaid/iotagent-node-lib project instead of `master` (where the stable, not the development version is) :)